api/tbcapi/tbcapi.go

```diff
@@ -101,14 +104,31 @@ type BalanceByAddressResponse struct {
 	Error   *protocol.Error `json:"error,omitempty"`
 }
 
+type UtxosByAddressRawRequest struct {
+	Address string `json:"address"`
+	Start   uint   `json:"start"`
+	Count   uint   `json:"count"`
+}
+
+type UtxosByAddressRawResponse struct {
+	Utxos []api.ByteSlice `json:"utxos"`
+	Error *protocol.Error `json:"error"`
```

Comment:
Error shoud be optional. This goes for all other Error' in JSON as well

Reply:
ah via `omit_empty`?

Reply:
Yes, please add `omitempty` to this error to be consistent with all other api packages.